src/ts/components/seekbar.ts

```diff
@@ -328,11 +330,15 @@ export class SeekBar extends Component<SeekBarConfig> {
     player.on(player.exports.PlayerEvent.TimeShift, onPlayerSeek);
     player.on(player.exports.PlayerEvent.TimeShifted, onPlayerSeeked);
 
-    this.onSeek.subscribe((sender) => {
-      this.isUserSeeking = true; // track seeking status so we can catch events from seek preview seeks
+    let isGroupPlaybackAPIAvailable = (player: PlayerAPI): player is ExtendedPlayerAPI => {
+      return !!(player as ExtendedPlayerAPI).groupPlayback;
+    };
 
-      // Notify UI manager of started seek
-      uimanager.onSeek.dispatch(sender);
+    let handleSeekEvent = () => {
+      // If group playback is present and user still seeking after 100ms then it might be a scrubbing.
+      if (isGroupPlaybackAPIAvailable(player) && player.groupPlayback.hasJoined() && this.isUserSeeking && !suspension) {
```

Comment:
My UI knowledge got a bit rusty over the last years but can't we use the `scrubbing` flag for this potentially? 🤔

Reply:
hmm `scrubbing` is set on `mousemove` and that might not be true after the timeout. But as now we always want to suspend we don't need it 😄

Reply:
Removed https://github.com/bitmovin/bitmovin-player-ui/pull/533/commits/e99a9f453c23778e6378521bbc64351b278ca211